invoke-rune.lic

```diff
@@ -0,0 +1,134 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#focus-rune
```

Comment:
❌ Should match script name

Reply:
I changed the script name and forgot this. :(